src/components/ConversationSettings/LobbySettings.vue

```diff
@@ -145,7 +145,7 @@ export default {
 				return ''
 			}
 			const date = new Date(this.lobbyTimer)
-			return t('spreed', 'Start time: {date}', { date })
+			return t('spreed', 'Start time: {time}', { time: date.toString() })
```

Comment:
```suggestion
			return t('spreed', 'Start time: {date}', { date: date.toString() })
```
To not create a new string they have to translate?